eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -6685,4 +6685,33 @@ private function createContentWithReverseRelations(array $drafts)
 
         return $contentWithReverseRelations;
     }
+
+    /**
+     * @covers \eZ\Publish\API\Repository\ContentService::LoadVersionInfoListByContentInfo
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testLoadVersionInfoListByContentInfo(): void
+    {
+        $folder1 = $this->createFolder(['eng-GB' => 'Folder1'], 2);
+        $folder2 = $this->createFolder(['eng-GB' => 'Folder2'], 2);
+
+        $versionInfoList = $this->contentService->loadVersionInfoListByContentInfo([
+            $folder1->contentInfo,
+            $folder2->contentInfo,
+        ]);
+
+        self::assertCount(2, $versionInfoList);
+
+        foreach ($versionInfoList as $versionInfo) {
+            $loadedVersionInfo = $this->contentService->loadVersionInfo(
+                $versionInfo->getContentInfo(),
+                $versionInfo->versionNo
+            );
+            self::assertEquals($loadedVersionInfo, $versionInfo);
+        }
+    }
```

Comment:
This is okay, but we're trying to retire legacy integration tests. Given there's a lot of them, any addition increases technical debt. It's fairly simple to add missing pieced to `IbexaKernelTestCase` so it works also for this old version of core.
See: https://github.com/ezsystems/ezplatform-kernel/compare/ibx-6017-implement-loadVersionInfoList...ibx-6017-test-using-ibexa-test-kernel

I added there base class `\Ibexa\Tests\Integration\Core\RepositoryTestCase` and test case `\Ibexa\Tests\Integration\Core\Repository\ContentService\LoadVersionInfoTest`, which is almost exact copy-paste of the test here. Notice that it's no longer one big ContentServiceTest class. That's because those big integration test cases have proven to be difficult to maintain. We need to start splitting them into groups.